web-ng/src/app/components/options-editor/options-editor.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+  Copyright 2020 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<form [formGroup]="optionsGroup">
+  <div fxLayout fxLayoutGap="15px">
+    <span class="dot"></span>
```

Comment:
What is this dot for? Having trouble visualizing what relates to in the specs.

Reply:
![image](https://user-images.githubusercontent.com/27938664/82788624-65b0ef00-9e2e-11ea-8e1a-aa631f355aa5.png)
This dot is css styling for options

Reply:
The "option" you're referring to is a possible value in a "select" UI element. These don't make sense here since this is the form designer, not the actual final form where options would be shown. We would normally show these if we wanted the user to select one of the options from a list. Here the user isn't selecting anything, they're defining the options to be shown on the form.

Reply:
Deleted my previous comment, apologies for the confusion. Could you kindly indent it further to leave room for the eventual drag widget, and adjust the color to grey (e.g. `mat-color($foreground, light)`